xDripAPS.py

```diff
@@ -156,35 +166,35 @@ def get(self):
             results_as_dict.append(result_as_dict)
 
         conn.close()
-    return results_as_dict
+        return results_as_dict
 
     def post(self):
 
         # Get hashed API secret from request
         try:
             request_secret_hashed = request.headers['Api_Secret']
-            print 'request_secret_hashed : ' + request_secret_hashed
+            xLog('request_secret_hashed : ' + request_secret_hashed)
         except:
-            print "Client didn't pass in Api-Secret header"
+            xLog("Client didn't pass in Api-Secret header")
             return 'Client didnt pass in Api-Secret header',500
 
         if api_secret:
             # Get API_SECRET environment variable
             env_secret_hashed = os.environ['API_SECRET']
-            print "We authenticated using environment variable API_SECRET"
+            xLog("We will authenticate using environment variable API_SECRET")
 
         elif api_secret_xDripAPS:
             #get API_SECRET_xDripAPS environment variable if needed
             env_secret_hashed = os.environ['API_SECRET_xDripAPS']
-            print "We will authenticate using environment variable API_SECRET_xDripAPS"
+            xLog("We will authenticate using environment variable API_SECRET_xDripAPS")
        
         # Authentication check
         if request_secret_hashed != env_secret_hashed:
```

Comment:
There is a bug here - 
You compare hash value with non-hash value
request secret is hashed while env_secret_hashed is actually the plain text and not hashed